resilience4j-spring-boot/src/main/java/io/github/resilience4j/bulkhead/monitoring/health/BulkheadHealthIndicator.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2019 lespinsideg
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.github.resilience4j.bulkhead.monitoring.health;
+
+import io.github.resilience4j.bulkhead.Bulkhead;
+import io.github.resilience4j.bulkhead.BulkheadConfig;
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.actuate.health.HealthIndicator;
+
+import java.util.Optional;
+
+/**
+ * A Spring Boot health indicators which adds the state of a Bulkhead and it's metrics to the health endpoints
+ */
+public class BulkheadHealthIndicator implements HealthIndicator {
```

Comment:
Do you think it makes sense to add a BulkheadHealthIndicator if the health state is always up? 
Metrics are already shown in the metrics endpoint.

Reply:
I removed `BulkheadHealthIndicator`.  Do you think it is better to show it as down state when a bulkhead is full?